### ISSUE TITLE
[LATEST] Button/Git Pull

### ISSUE THREAD

**Issue by cauwelsdev** [lordkator/ZonamaDev#112]
@karlbunch 
Can't pull latest after a VM restart/close. Keeps on saying:
```
latest>> Host key verification failed.
latest>> fatal: Could not read from remote repository.
```

**Comment by karlbunch** [comment]
Closed by 19bd79608015169add6b753d423c19399873a746

Feel free to test latest and re-open if you have the same behavior.

**Comment by cauwelsdev** [comment]
working as intended now.
